docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1798 @@
+{
```

Comment:
<p>I'm not  completely sure about this since it is not cumulative probability, right? I think these are the probabilities of making it to K + 1 years given that you made it to K years. </p><p> </p><p>Also, it makes sense, since the thresholds are not sorted anymore (you can see that in the mean in the summary above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Correct, it is not a cumulative probability. Ahhhh. I understand it now. For some reason this just now clicked for me.</p>